src/main/java/sjchat/restapi/Application.java

```diff
@@ -0,0 +1,12 @@
+package sjchat.restapi;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
+
+  public static void main(String[] args) {
+    SpringApplication.run(Application.class, args);
```

Comment:
Are you only using Spring for the rest API?

It seems a bit... heavyweight? Surely there must be less intrusive HTTP service frameworks that don't require the entire spring framework.